<launch>
	<include file="$(find-pkg-share openni2_camera)/launch/camera_only.launch.py"/>
  <node pkg="rgbd_slam" 		exec="rgbd_slam_node" name="rgbd_slam_node" output="screen">
		<param name="rgb_topic" 	     				value="/camera/rgb/image_raw" />
		<param name="depth_topic" 	     			value="/camera/depth/image_raw" />
		<param name="calibration_file" 	 			value="$(find-pkg-share rgbd_calibration)/config/rgbd_calibration.txt" />

		<!--Feature detection
		<param name="horizontal_tiles_masks" 	type="int" 	value="4" />
		<param name="vertical_tiles_masks" 		type="int" 	value="3" />
		<param name="gauss_filter_size" 			type="int" 	value="5" />
		<param name="rgb_im_width" 						type="int" 	value="640" />
		<param name="rgb_im_height" 					type="int" 	value="480" />
		<param name="min_features_to_init" 		type="int" 	value="70" />
		<param name="min_features_in_mask_to_detect_new" 		type="int" 	value="5" />
		<param name="max_optical_flow_error" 								type="double" 	value="5" />
		<param name="min_features_in_mask_distribution" 		type="int" 	value="3" />
		<param name="min_feature_distribution" 							type="double" 	value="0.4" />
		<param name="min_features_to_track_only" 						type="int" 	value="50" />-->
	</node>
</launch>